install_files/ansible-base/roles/restore/tasks/perform_restore.yml

```diff
@@ -23,8 +23,10 @@
   connection: local
   become: no
   unarchive:
+    # Avoid creating an unnecessary local copy
+    remote_src: yes
     dest: "{{ torrc_check_dir.path }}/backup/"
-    src: "{{ restore_file }}"
+    src: "{{playbook_dir}}/{{ restore_file }}"
```

Comment:
this is unrelated to the PR, I wonder if a rebase went awry somewhere. will take a look.

Reply:
Fixed preparing the pull request for integration on current develop branch